cumulusci.yml

```diff
@@ -468,8 +484,27 @@ flows:
                 flow: config_default
             4:
                 task: create_community_users
+            # 5:
+            #     flow: add_nmg_permissions
+
+    qa_org:
+        steps:
+            1:
+                flow: None
+            2:
+                flow: None
+            3:
+                flow: None
+            4:
+                task: None
```

Comment:
@Julian88Tex  I missed the meeting but curious on why we need this, can u please explain

Reply:
@skristem sure basically this is currently the only way to override steps in an existing flow. It's discussed in more detail in this slack posts's thread: https://sfdotp.slack.com/archives/C010F8UNC75/p1589920858486600

@davisagli also let me know there is a feature request for this to be changed in cci: https://github.com/SFDO-Tooling/CumulusCI/issues/1098